IDE/MDK5/dummy.c

```diff
@@ -0,0 +1,16 @@
+#include <wolftpm/tpm2.h>
+
+void* TPM2_IoGetUserCtx(void){
+ return NULL;
+}
+
+int   TPM2_IoCb(TPM2_CTX* ctx, const byte* txBuf, byte* rxBuf,
+word16 xferSz, void* userCtx){
+	(void) ctx;
+	(void) txBuf;
+	(void) rxBuf;
+	(void)xferSz;
+	(void)userCtx;
+	return 0;
+}
+	
```

Comment:
@kojo1, if you'd like to plug in the STM32 CubeMX HAL SPI code its here:

https://github.com/wolfSSL/wolfTPM/blob/master/examples/tpm_io.c#L34
https://github.com/wolfSSL/wolfTPM/blob/master/examples/tpm_io.c#L60
https://github.com/wolfSSL/wolfTPM/blob/master/IDE/OPENSTM32/Src/main.c#L210